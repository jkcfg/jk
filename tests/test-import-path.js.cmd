jk run -I ./import-paths-files import-paths.js
