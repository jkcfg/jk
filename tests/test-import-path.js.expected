value from module on module path
